src/ui/gui.css

```diff
@@ -118,7 +123,6 @@ table.spaced {
 
 table.dice {
     padding: 0px;
-    spacing: 0px;
```

Comment:
I'm pretty sure I removed this because there is no CSS tag called "spacing". It's possible that NetBeans was giving me a warning to that effect, but I can't remember exactly.